src/styles/components/_pane.scss

```diff
@@ -94,6 +97,7 @@
     padding: var(--neeto-ui-pane-body-padding-y) var(--neeto-ui-pane-body-padding-x);
     font-size: var(--neeto-ui-pane-body-font-size);
     overflow-y: auto;
+    flex-grow: 1;
```

Comment:
@goutham-subramanyam _a Revert the change to the Pane layout for now, until we confirm it doesn't affect existing products.